src/commands/sensor/aggregates/sensor.aggregate.ts

```diff
@@ -114,12 +122,13 @@ export class SensorAggregate extends Aggregate {
 
   private onSensorOwnershipTransferred(eventMessage: EventMessage) {
     const event: SensorOwnershipTransferred = eventMessage.data as SensorOwnershipTransferred;
-    this.logger.debug(`Not implemented: aggregate.eventHandler(${event.constructor.name})`);
+    this.state.ownerIds = this.state.ownerIds.filter((ownerId) => ownerId !== event.oldOwnerId);
+    this.state.ownerIds.push(event.newOwnerId);
   }
 
   private onSensorOwnershipShared(eventMessage: EventMessage) {
     const event: SensorOwnershipShared = eventMessage.data as SensorOwnershipShared;
-    this.logger.debug(`Not implemented: aggregate.eventHandler(${event.constructor.name})`);
+    this.state.ownerIds = this.state.ownerIds.concat(event.ownerIds);
```

Comment:
Op line 126 gebruik je `push` en hier `concat` ... deze doen hetzelfde, niet? Ze geven alleen verschillende dingen terug; zie [dit antwoord](https://stackoverflow.com/a/44572132/3811235) ... 🤔

Reply:
yes klopt, dat ik expres, in de share ownership api kan je een lijst van owners meegeven met wie de sensor gedeelt wordt.